js/index.js

```diff
@@ -0,0 +1,159 @@
+}
+
+const mainProgramHeader = document.querySelector('.heading-program');
+const programContainer = document.createElement('div');
+programContainer.classList.add('card-program-container');
+
+programCard.forEach((program) => {
+  card += contentCard(program);
+});
+
+programContainer.insertAdjacentHTML('afterbegin', card);
+mainProgramHeader.after(programContainer);
+
+// Featured Speaker
+
+const speakerCard = [
+  {
+    id: 0,
+    name: 'David Kau',
+    image: 'david-kau.jpg',
+    role: 'Comedian',
+    details: 'Focusing on a collaborative approach in a networked environment, he created the concept of co-production based on sharing, such as open source software and Wikipedia.',
+  },
+  {
+    id: 1,
+    name: 'Ntsudeni Ndou',
+    image: 'trevor-gumbi.jpg',
+    role: 'Comedian',
+    details: 'By developing Asia&apos;s first Internet protocol network SDN and leading Korea&apos;s first private line Internet connection in 1990, it ushered in the era of the Internet in earnest.',
+  },
+  {
+    id: 2,
+    name: 'Trevor Noah',
+    image: 'trevor-noah.jpg',
+    role: 'Comedian',
+    details: 'European integration and online youth participation in politics and democracy are major concerns, and a report has been published that will potentially affect the revision of the EU&apos;s copyright law in July.',
+  },
+  {
+    id: 3,
+    name: 'Loyiso Gala',
+    image: 'loyiso-gala.jpg',
+    role: 'Comedian',
+    details: 'Layla Tretikov is the general secretary of the Wikimedia Foundation, a non-profit organization that runs Wikipedia. Wikipedia is provided free of charge in 290 languages every month to over 100 million people, nearly half of the world&apos;s population',
+  },
+];
+
+function contentSpeakers(speaker) {
+  const cards = `
+    <div class="speaker-holder">
+        <div class="speaker-img-holder">
+            <img src="./assets/${speaker.image}" alt="${speaker.image}" class="speaker-img">   
+        </div>
+        <div >
+        <h3 class="speaker-title">${speaker.name}</h3>
+        <h4 class="speaker-role underline-role">${speaker.role}</h3>
+        <p class="speaker-details">${speaker.details}</p>
+        </div>
+
+   </div>
+    `;
+  return cards;
+}
+
+let cardSpeaker = '';
+
+speakerCard.forEach((speaker, index) => {
+  if (index < 2) {
+    cardSpeaker += contentSpeakers(speaker);
+  }
+});
+
+const heading = document.querySelector('.heading-speakers');
+const speakerContainer = document.createElement('div');
+speakerContainer.classList.add('speaker-container');
+speakerContainer.insertAdjacentHTML('afterbegin', cardSpeaker);
+
+heading.after(speakerContainer);
+
+let moreCardSpeaker = '';
+const moreSpeaker = document.getElementById('more-speakers');
+
+moreSpeaker.addEventListener('click', () => {
+  speakerCard.forEach((speaker, index) => {
+    if (index >= 2) {
+      moreCardSpeaker += contentSpeakers(speaker);
+    }
+  });
+  speakerContainer.insertAdjacentHTML('afterbegin', moreCardSpeaker);
+
```

Comment:
[OPTIONAL]
- [ ] The `see_more` feature was supposed to be visible only on the mobile version. 
On the desktop, all the speakers are supposed to be visible

Reply:
thanks @christianbiring1 , will work on that